charts/kubescape-operator/templates/_helpers.tpl

```diff
@@ -11,6 +11,9 @@ submit: {{ $submit }}
     {{- if empty .Values.account -}}
       {{- fail "submitting is enabled but value for account is not defined: please register at https://cloud.armosec.io to get yours and re-run with  --set account=<your Guid>" }}
     {{- end -}}
+    {{- if empty .Values.token -}}
```

Comment:
Let's wait for this statement. 
I think we should not force the token in our first iteration, mainly because you are creating a dependency here by releasing the support in the BE.